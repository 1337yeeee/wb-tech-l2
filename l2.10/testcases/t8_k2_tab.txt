id1	A
id2	C
id3	B
