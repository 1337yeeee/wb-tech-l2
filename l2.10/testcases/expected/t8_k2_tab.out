id1	A
id3	B
id2	C
